src/core/JsonConfigCommandCenter.ts

```diff
@@ -11,6 +11,7 @@ import { JSONEditorMode } from "jsoneditor";
 export class JsonConfigCommandCenter {
     public static titleUpdateCallback: (text: string, mode: JSONEditorMode) => void;
     public static getOriginalJsonConfig: () => any;
+    public static hasErrors: (error?: boolean) => boolean;
```

Comment:
i still don't understand why this has to be a callback. why can't this be just a boolean property?

Reply:
Updated to Boolean property